tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp

```diff
@@ -593,6 +593,7 @@ void test_2d_orient_variables_on_slice() {
 
 }  // namespace
 
+// [[Timeout, 10]]
```

Comment:
I think this needs to be `TimeOut`

Reply:
Wait, unclear... Documentation and code differ :P

Reply:
I copied this from somewhere, so it might be case insensitive

Reply:
Added a commit that just makes these consistent 🤷‍♂️

Reply:
Yea, at some point we did the smart thing and made it case insensitive :shrug: